src/tests/CMakeLists.txt

```diff
@@ -30,9 +30,3 @@ endif()
 
 serac_add_tests( SOURCES       axom_smoketest.cpp
                  DEPENDS_ON    axom::inlet)
```

Comment:
Why was this moved?

Reply:
Since it depends on `serac_infrastructure`, i thought it made more sense to move the test to `src/serac/infrastructure/tests`

Reply:
Ah, I didn't look at the source file. It actually only depends on `axom` for the logging (which could easily be removed`. I recommend we move it back and swap the `serac_infrastructure` for `axom`.

TPL smoke tests should where it was originally.

Reply:
ended up needing to include `axom::inlet` but its been moved